app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -53,6 +53,7 @@ def index
       end
 
       def show
+        raise ActionController::RoutingError, "Not Found" unless set_proposal
```

Comment:
Better `@proposal.present?`

Reply:
It is just as it is in `ProposalsController` of Original Decidim. Check the next url.
https://github.com/decidim/decidim/blob/d94c117fd1dc890ed40084010190971c7d101fed/decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb#L56

Reply:
if you think it is better to leave it as the original then ok